app/src/main/java/com/keylesspalace/tusky/pager/ImagePagerAdapter.kt

```diff
@@ -0,0 +1,28 @@
+package com.keylesspalace.tusky.pager
+
+import android.support.v4.app.Fragment
+import android.support.v4.app.FragmentManager
+import android.support.v4.app.FragmentStatePagerAdapter
+
+import com.keylesspalace.tusky.entity.Attachment
+import com.keylesspalace.tusky.fragment.ViewMediaFragment
+
+import java.util.Locale
+
+class ImagePagerAdapter(fragmentManager: FragmentManager, private val attachments: List<Attachment>, private val initialPosition: Int) : FragmentStatePagerAdapter(fragmentManager) {
```

Comment:
You know how to format it :wink:

Reply:
Sorry then!
https://kotlinlang.org/docs/reference/coding-conventions.html#class-header-formatting